proofzero/rollupid
# feat(passport): Inclusion of magic links for email OTPs

### Description

Create a new route that makes a post to verify hooking into existing mail code verification mechanism and authentication process

### Related Issues

- Closes #2707 

### Testing

- [x] Added a console log for magic link, used that to:
1. Create new account
2. Connect to existing account
3. Join a group
- [x] Manually added bad code to test that verify screen with error is presented
- [ ] Dev deployment & visual check

### Checklist
